### ISSUE TITLE
Does not work in accessibility mode.

### ISSUE THREAD

**Issue by jthurston** [wnielson/sencha-SlideNavigation#34]
In my environment, all applications must past 508 testing (accessibility) the button does not work if you put an iPhone into accessibility screen reader mode.

**Comment by wnielson** [comment]
Thanks for the report.
